src/acbm/config.py

```diff
@@ -17,19 +17,42 @@ class Parameters(BaseModel):
     boundary_geography: str
 
 
+@dataclass(frozen=True)
+class MatchingParams(BaseModel):
+    load_hh: bool | None = False
+    load_ind: bool | None = False
+    # Define required columns for matching
+    required_columns: list[str] | tuple[str] = (
```

Comment:
shouldn't we specify the `required_columns` and `optional_columns` in the toml file instead of here?